hypothesis-python/src/hypothesis/internal/conjecture/dfa/__init__.py

```diff
@@ -46,6 +65,10 @@ def transition(self, i, c):
         character c from a string."""
         raise NotImplementedError()
 
+    @property
+    def alphabet(self):
+        return range(256)
+
     def transitions(self, i):
```

Comment:
I didn't notice earlier, but this should be named `live_transitions`, or something along those lines.

Reply:
I kinda like the name `transitions` and would be reluctant to change it. It's kinda intending to capture the fact that dead nodes might as well not be there - a missing transition isn't really interestingly different to a transition to a dead node.